Cargo.toml

```diff
@@ -27,6 +30,11 @@ lazy_static = "1.4.0"
 log = "0.4"
 ndarray = "0.15.6"
 ndarray-stats = "0.5.1"
-rdkafka = { version = "0.31.0", features = ["cmake-build", "ssl", "gssapi", "sasl"] }
+rdkafka = { version = "0.31.0", features = [
```

Comment:
Was there a reason for this? It would be preferable to keep the one line per dependency format.

Reply:
This was automatically formatted when I saved the file for some reason. I'm not sure whether it would be the LSP doing this or `treefmt`.